cmd/podman/generate/systemd.go

```diff
@@ -56,8 +59,17 @@ func init() {
 	flags.BoolVarP(&files, "files", "f", false, "Generate .service files instead of printing to stdout")
 	flags.BoolVar(&systemdOptions.TemplateUnitFile, "template", false, "Make it a template file and use %i and %I specifiers. Working only for containers")
 
-	flags.UintVarP(&systemdTimeout, timeFlagName, "t", containerConfig.Engine.StopTimeout, "Stop timeout override")
-	_ = systemdCmd.RegisterFlagCompletionFunc(timeFlagName, completion.AutocompleteNone)
+	flags.UintVarP(&startTimeout, startTimeoutFlagName, "", 0, "Start timeout override")
+	_ = systemdCmd.RegisterFlagCompletionFunc(startTimeoutFlagName, completion.AutocompleteNone)
+
+	// NOTE: initially, there was only a --time/-t flag which mapped to
+	// stop-timeout. To remain backwards compatible create a hidden flag
+	// that maps to StopTimeout.
+	flags.UintVarP(&stopTimeout, stopTimeoutFlagName, "", containerConfig.Engine.StopTimeout, "Stop timeout override")
+	_ = systemdCmd.RegisterFlagCompletionFunc(stopTimeoutFlagName, completion.AutocompleteNone)
+	flags.UintVarP(&stopTimeout, stopTimeoutCompatFlagName, "t", containerConfig.Engine.StopTimeout, "Backwards alias for --stop-timeout")
```

Comment:
Sorry, I forgot to say this yesterday. You should make this an alias instead of a hidden flag, see https://github.com/containers/podman/blob/1bfbb28b0365790552483b961b4bd48a69dd8070/cmd/podman/system/unshare.go#L38-L46

Reply:
Why?  I don't want `--time` to be displayed anymore.

Reply:
```
$ ./bin/podman generate systemd test --time 1234
Error: unknown flag: --time
```

Reply:
Aaaah, nevermind, I see what's wrong now.

Reply:
How shall we deal with `-t`?  I am unable to catch that in the SetNormalizedFunc.

Reply:
Ah I didn't knew that it cannot handle short flag names...
I think we have to stick with your current version, sorry for wasting your time on this.

Reply:
No worries at all.  It was a useful exercise :+1: Thanks for checking!